lib/v3/error_response.rb

```diff
@@ -0,0 +1,10 @@
+class ErrorResponse
+  attr_accessor :error_identifier, :default_message
+
+  ERRORS = [:login_merge_newSchoolNameMissing].freeze
```

Comment:
🙏

Reply:
How are we going to document these?

Reply:
#780 should handle this, if not auto generated we'll add manually.